content/news/XXXX-XX-XX-mixxx-2_5-released.md

```diff
@@ -0,0 +1,458 @@
+macOS versions earlier than 11 can have a "Last Christmas".
+Windows versions earlier than version 10 build 1809 can be "Driving home for Christmas".
+For Ubuntu versions earlier than 22.04 "Christmas Will Break Your Heart".
+By stepping over from Qt5 to Qt6 Mixxx is forced to increase the minimum system requirements to match those mandated by Qt.
+
+That's why the support has been dropped for macOS earlier than 11, Windows versions earlier then version 10 build 1889 and Ubuntu earlier then 22.04.
+
+#### Let's cheer for the elves
+
+Ho Ho Ho, our sleigh is filled with many shiny presents:
+
+* A red box filled with improvements to waveforms: 'SlipMode waveform' gives you visual aid
+while scratchin'
+
+@Video(https://www.youtube.com/watch?v=6VRz48Ler9o)
+
+* The 'beats until next marker' helps you to never miss the break
+
+@Video(https://www.youtube.com/watch?v=uYCXhKeJVFo)
+
+* The GL ES support makes the waveforms run smooth.
+
+@Video(https://www.youtube.com/watch?v=5VJBuUR_cn8)
+
+* With a nice yellow bow around it, the new Skin & Interface features: the menubar can be hidden,
+reworked toggles (fullscreen, cue popup, vinyl control), improved tooltips, improved controls and checks (AutoDJ,
+effects), new command line interface options...
+
+@Video(https://www.youtube.com/watch?v=qMRcJvntboE)
+
+* Our engine became a V8 sleigh: undo function for BPM/beats, beatloop anchor, rate tap button...
+
+@Video(https://www.youtube.com/watch?v=7XKGodjiTNg)
+
+* "Rudolph, the Red-Nosed Reindeer" will sound much better with the compressor and glitch effects and the improved backend, which now supports also native Audio Unit (AU) effect plugins on macOS.
+
+
+@Video(https://www.youtube.com/watch?v=klOjBuZm0Qg)
+
+* Your "Jingles" and "Bells" can be sorted and found easier with the new library features: Cut, Copy and Paste in tracklists.
+
+@Video(https://www.youtube.com/watch?v=2qPXCCaGVGI)
+
+* Tracks can be moved with ALT+Up/Down/PgUp/PgDn
+
+@Video(https://www.youtube.com/watch?v=MVvEP18a0xw)
+
+* With a summary containing the number of tracks and the total duration added to the name of crates, playlists and historylists makes it easy to find the right list.
+
+@Video(https://www.youtube.com/watch?v=0HtpOOIpjaM)
+
+* The searchengine received new BPM filters, an 'OR'-operator, new import possibilities,
+custom track colors for missing or played tracks ...
+
+@Video(https://www.youtube.com/watch?v=qfnH7jWw0iY)
+
+* "Santa Tell Me" what's new in preferences: some improved layout, the fetching of the soundcard's sample rate is fixed,
+a new setting for 'first hotcue', multiple options can be selected in the MIDI input editor...
+
+* Many "Days Of Christmas" full of improved Controller Mappings for Denon MC7000, Numark Scratch, Pioneer DDJ-FLX4, Traktor Kontrol S4 MK3 and MIDI for light.
+
+* A Controller Backend as a "Fairytale of New York" to map your controller: sysex, deck's track menu, support for bulk devices,
+drop of lodash dependency...

```

Comment:
Can you try to indent all lines but the first?
that might fix the bullet point offset seen here
![image](https://github.com/user-attachments/assets/1a379190-67c0-450a-8b18-b27116b5a32e)
```suggestion
* "Santa Tell Me" what's new in preferences: some improved layout, the fetching of the soundcard's sample rate is fixed,
  a new setting for 'first hotcue', multiple options can be selected in the MIDI input editor...

* Many "Days Of Christmas" full of improved Controller Mappings for Denon MC7000, Numark Scratch, Pioneer DDJ-FLX4, Traktor Kontrol S4 MK3 and MIDI for light.

* A Controller Backend as a "Fairytale of New York" to map your controller: sysex, deck's track menu, support for bulk devices,
  drop of lodash dependency...

```

Reply:
did not help

Reply:
And there mustn't be blank lines between the bullet points

Reply:
Can you try to indent all lines but the first?
that might fix the bullet point offset seen here
![image](https://github.com/user-attachments/assets/1a379190-67c0-450a-8b18-b27116b5a32e)
```suggestion
* "Santa Tell Me" what's new in preferences: some improved layout, the fetching of the soundcard's sample rate is fixed,
a new setting for 'first hotcue', multiple options can be selected in the MIDI input editor...

* Many "Days Of Christmas" full of improved Controller Mappings for Denon MC7000, Numark Scratch, Pioneer DDJ-FLX4, Traktor Kontrol S4 MK3 and MIDI for light.

* A Controller Backend as a "Fairytale of New York" to map your controller: sysex, deck's track menu, support for bulk devices,
  drop of lodash dependency...

```

Reply:
Hi Jan,  that indeed reesolves the bullet <new line> text, but you asked to add some blank lines to separate the headlines but ... now the last headlines are a wall of texts again (your words)

Reply:
I meant blank lines to separate the text into paragraphs (rather than one paragraph).

Blank lines between bullet points separates the list into multiple lists, so we can't add blank lines there (unless you remove the bullet points `* ` and use regular text paragraphs).

I'll make a suggestion to show you what I mean.